components/tooltip/script.js

```diff
@@ -225,28 +225,46 @@ class Tooltip extends HTMLElement {
      * easily deconstruct the value like this: let [orientation, position] = getPositionCoords(...).
     */
     getPositionCoords(orientation, elementSize) {
-        const tooltipSize = this.getBoundingClientRect();
-
         const elementPosition = {
-            top: (elementSize.top + elementSize.height / 2) - tooltipSize.height / 2,
-            left: elementSize.left + (elementSize.width / 2) - tooltipSize.width / 2,
+            top: (elementSize.top + elementSize.height / 2),
+            left: elementSize.left + (elementSize.width / 2),
         };
 
+        this.style.transform = 'translate(-50%, -50%)';
+
+
+        return this.setOrientationAndPosition(orientation, elementPosition, elementSize);

```

Comment:
This is my bad on the naming, it should be getOrientationAndPosition. I added it as a separate function because the switch made the linter trigger for the 'Too long function' rule